### PR TITLE
add 'ta config path'

### LINKED ISSUE #138: Make a `task config` command to dump the configuration
..including defaults.

### PR DESCRIPTION
**Pull request by djmitche**
Fixes #138

### CODE REVIEW

File: cli/src/invocation/cmd/config.rs [hunk 1]
```diff
@@ -19,6 +19,13 @@ pub(crate) fn execute<W: WriteColor>(
             writeln!(w, "{:?}.", filename)?;
             w.set_color(ColorSpec::new().set_bold(false))?;
         }
+        ConfigOperation::Path => {
+            if let Some(ref filename) = settings.filename {
+                writeln!(w, "{}", filename.to_string_lossy())?;
+            } else {
+                return Err(anyhow::anyhow!("No confiuration filename found").into());
```
Comment (dbr) [inline cli/src/invocation/cmd/config.rs:26]: Typo in `configuration`

**Review by dbr — CHANGES_REQUESTED** [review]
Looks good but for the typo!

**Comment by dbr** [comment]
BTW I've not played around with it yet but I found https://docs.rs/camino/1.0.4/camino/ which seems interesting alternative to `to_string_lossy` and lots of `Result<..>` handling with file paths

> camino is an extension of the std::path module [...] like the standard library’s PathBuf and Path types, except they are guaranteed to only contain UTF-8 encoded data

**Comment by djmitche** [comment]
I'm not terribly worried about the `_lossy` bit.  I suspect the number of people who will be using non-utf8 filenames with taskchampion (which doesn't do much with files) is approximately zero :)

**Pull request merged**